src/test/java/project/web/GasStationControllerTest.java

```diff
@@ -60,12 +59,13 @@ void getGasPriceInfoTestWithValidInfo() {
         GasPriceInfoDTO gasPriceInfo = createTestGasPriceInfoDTO();
         when(gasStationService.getGasPriceInfo(fuelType)).thenReturn(gasPriceInfo);
 
-        // WHEN
-        ResponseEntity<GasPriceInfoDTO> response = gasStationController.getGasPriceInfo(fuelType);
-
-        // THEN
-        assertEquals(HttpStatus.OK, response.getStatusCode());
-        assertEquals(gasPriceInfo, response.getBody());
+        // WHEN THEN
```

Comment:
here great example, why you should keep the WHEN separated - in this case you have duplications!

Reply:
fixed